cpp/src/dictionary/detail/concatenate.cu

```diff
@@ -169,16 +169,10 @@ struct dispatch_compute_indices {
     return result;
   }
 
-  template <typename Element>
+  template <typename Element, typename... Args>
   typename std::enable_if_t<!cudf::is_relationally_comparable<Element, Element>(),
                             std::unique_ptr<column>>
-  operator()(column_view const&,
-             column_view const&,
-             column_view const&,
-             offsets_pair const*,
-             size_type const*,
-             rmm::cuda_stream_view stream,
-             rmm::mr::device_memory_resource*)
+  operator()(Args&&...)
   {
     CUDF_FAIL("list_view as keys for dictionary not supported");
```

Comment:
is this the right error message?

Reply:
See other comment about `is_relationally_comparable`